pkg/cache/v2/simple.go

```diff
@@ -119,7 +132,12 @@ func (cache *snapshotCache) SetSnapshot(node string, snapshot Snapshot) error {
 				if cache.log != nil {
 					cache.log.Debugf("respond open watch %d%v with new version %q", id, watch.Request.ResourceNames, version)
 				}
-				cache.respond(watch.Request, watch.Response, snapshot.GetResources(watch.Request.TypeUrl), version)
+				cache.respond(
```

Comment:
Probably not worth the change since its functionally equivalent and doesn't seem to violate linting).